man/man1/git-town-append.1

```diff
@@ -0,0 +1,44 @@
+.TH "GIT-TOWN-APPEND" "1" "12/26/2016" "Git Town 2\&.1\&.0" "Git Town Manual"
+
+.SH "NAME"
+git town-append \- create a new feature branch as a direct child of the current branch
+
+
+.SH "SYNOPSIS"
+\fIgit town-append\fR <branch_name>
+.br
+\fIgit town-append\fR (--abort | --continue)
+
+
+.SH "DESCRIPTION"
+
+.PP
+Syncs the given branch,
```

Comment:
Fixed to `current branch`. What do you think, should it sync the current branch or go straight to creating the new branch?